.vscode/settings.json

```diff
@@ -1,9 +1,7 @@
 {
     "files.exclude": {
         ".husky": true,
-        ".vscode": true,
         "dist": true,
-        "node_modules": true
```

Comment:
I think these are helpful to see in the editor - often enough we have to touch stuff that is from the prose mirror npm packages - not going to go hunting in folder explorer to find/browse/view it.  But curious if there was a reason why removing any of these from the editor view made sense.

Reply:
Looks like the yeas have it - removing folder excludes from project settings - but still configurable at user level